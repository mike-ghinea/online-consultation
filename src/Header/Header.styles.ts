import styled from "styled-components";

const HeaderWrapper = styled.header`
  display: flex;
  padding: 1rem;
  background-color: rgb(0, 63, 130);
  color: white;
  justify-content: center;
  @media (min-width: 480px) {
    padding: 1rem 1rem 1rem 3rem;
    justify-content: flex-start;
  }
`;

const Logo = styled.img`
  height: 50px;
`;
export default { HeaderWrapper, Logo };
