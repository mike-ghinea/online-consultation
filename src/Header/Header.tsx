import s from "./Header.styles";
import meLogo from "../assets/me_logo.png";
const Header: React.FC = () => {
  return (
    <s.HeaderWrapper>
      <s.Logo src={meLogo} alt="MedExpress logo" />
    </s.HeaderWrapper>
  );
};

export default Header;
