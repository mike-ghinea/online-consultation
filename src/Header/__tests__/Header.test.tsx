import Header from "../Header";
import { render } from "@testing-library/react";

describe("Header", () => {
  it("should render correctly", () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toMatchSnapshot();
  });
});
