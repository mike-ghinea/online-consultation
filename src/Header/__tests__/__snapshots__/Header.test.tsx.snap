// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<body>
  <div>
    <header
      class="sc-blHHSb jzQQDl"
    >
      <img
        alt="MedExpress logo"
        class="sc-gtLWhw jKGAVT"
        src="[object Object]"
      />
    </header>
  </div>
</body>
`;
